python/lsst/ts/standardscripts/maintel/m1m3/check_actuators.py

```diff
@@ -86,15 +88,18 @@ async def assert_feasibility(self):
                 setattr(self.mtcs.check, comp, False)
 
         # Check all enabled, liveliness and the m1m3 detailed state
+        self.expected_states = {DetailedState.PARKED, DetailedState.PARKEDENGINEERING}
+        self.detailed_state = await self.mtcs.assert_m1m3_detailed_state(
```

Comment:
This method does not return the detailed state. It asserts that m1m3 is in the expected states and raises an exception if they are not.

If you want a custom error message you need to do something like this:

```
try:
    await self.mtcs.assert_m1m3_detailed_state(expected_states)
except AssertionError:
    raise RuntimeError(
        f"M1M3 detailed should be in one of the following detailed states: {detailed_states}."
        "Lower M1M3 before continuing."
)
```

I would probably do this after the other checks though. Because if m1m3 is not alive or not in enabled state this will also be false.

Reply:
Yes, of course :(